src/core/data/particles/particle_utilities.h

```diff
@@ -33,6 +33,60 @@ namespace core
         }
```

Comment:
might aswell change this file to "namespace PHARE::core {"

while you're at it?